### DISCUSSION TITLE
Can I force kitty to bold a font even if it dosen't have a style for it?

### DISCUSSION THREAD

**Discussion by esrevinue** [kovidgoyal/kitty#7790] [Q&A] [answered]
Hello!

I _really_ like the IBM 3270 font provided by the Powerline Fonts github, but it doesn't have included bold or italic styles. On the foot emulator, this isn't a problem, and it will make bold and italicise the font with no questions asked. Kitty, on the other hand, seems to require there to be the specific family available for the chosen font to work with such features. Is there any way to force kitty to bold/italicise font? Attached below are screenshots comparing the two.

Thanks in advance for any help :)
![2024-27-08](https://github.com/user-attachments/assets/d88e3a42-6496-47a7-af0b-333c40e0e5ac)

**Comment by kovidgoyal** [comment, marked as answer]
No, kitty does not do fake bold/italic.

**Reply by esrevinue** [reply]
Thank you for your quick response. Are there any plans to add this feature in the future?

**Comment by kovidgoyal** [comment]
no, no plans
